cypress/e2e/style-selection.cy.ts

```diff
@@ -14,7 +14,7 @@ describe('Style selector', () => {
   })
 
   it('has no style value in localStorage', () => {
-    expect(localStorage.getItem('basemap_2015_global')).to.eq('')
+    expect(localStorage.getItem('basemap_2015_global') || '').to.eq('')
```

Comment:
It's like testing if true is equal to true. Change the value of .to.eq('') instead.

Reply:
no it's not like testing true, it's  testing for null or '' if there is a value, the test fails.

I improved the code, to unregister a style, the whole key is removed instead of setting '', thanks for the comment